docs/content/getting-started/production-best-practices.mdx

```diff
@@ -16,15 +16,15 @@ import {
 
 The following list outlines some guidelines and best-practices for running OpenFGA in a production environment:
 
-- [Configure Authentication](./setup-openfga.mdx#configuring-authentication)
+- [Configure Authentication](./setup-openfga/docker#configuring-authentication)
```

Comment:
```suggestion
- [Configure Authentication](./setup-openfga/docker.mdx#configuring-authentication)
```

Please add in the mdx suffix to ensure the link validator detects the correct link